src/Smalot/PdfParser/PDFObject.php

```diff
@@ -224,6 +224,65 @@ private function formatContent(?string $content): string
             return '';
         }
 
+        // Find all inline image content and replace them so they aren't
+        // affected by the next steps
+        $pdfInlineImages = [];
+        $offsetBI = 0;
+        while (preg_match('/\sBI\s(\/.+?)\sID\s(.+?)\sEI(?=\s|$)/s', $content, $text, \PREG_OFFSET_CAPTURE, $offsetBI)) {
+            // Attempt to detemine if this instance of the 'BI' command
+            // actually occured within a (string) using the following
+            // steps:
+
+            // Remove any escaped parentheses from the alleged image
+            // characteristics data
+            $para = str_replace(['\\(', '\\)'], '', $text[1][0]);
+
+            // Remove all correctly ordered and balanced parentheses
```

Comment:
I am only nitpicking here: could you add a numbering to the steps you refering here? I assume the comment in line 236 is for step 1 and comment in line 240 for step 2?